### PR TITLE
feature: S3C-4091-getAccount

### PR DESCRIPTION
**Pull request by anurag4DSB**
Vault client option for getAccount Vault API.

### CODE REVIEW

**Review by ilkescality — APPROVED** [review]

File: bin/vaultclient [hunk 1]
```diff
@@ -146,6 +146,23 @@ program
...
+    .option('--account-name <Account Name>')
+    .option('--account-id <ACCOUNT ID')
+    .option('--canonical-id <CANONICAL ID>')
+    .option('--email <EMAIL>')
+    .action(action.bind(null, 'get-account', (client, args) => {
+        client.getAccount({
+            accountArn: args.accountArn || undefined,
+            accountName: args.accountName || undefined,
+            accountId: args.accountId || undefined,
+            canonicalId: args.canonicalId || undefined,
```
Comment (saitharunthodupunuru) [inline bin/vaultclient:161]: is canonicalId going to be same as `cd_tenant_id` as we would like to `getAccount` with `cd_tenant_id` where Account has List `cd_tenant_ids`.

Other idea could be use listAccounts and filter using `cd_tenant_id`.

Which approach would be better do you think?

**Review by saitharunthodupunuru — CHANGES_REQUESTED** [review]

File: bin/vaultclient (hunk 1 above)
Comment (anurag4DSB) [reply bin/vaultclient:161]: cd_tenant_id is out of the scope of this ticket as it has not been implemented in Account yet.

**Review by saitharunthodupunuru — APPROVED** [review]

**Pull request merged**